src/components/ClassificationPanel.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import {
+  Platform,
+  TouchableOpacity,
+  View
+} from 'react-native';
+import EStyleSheet from 'react-native-extended-stylesheet'
+import StyledText from './StyledText'
+
+const topPadding = (Platform.OS === 'ios') ? 10 : 0
+
+class ClassificationPanel extends Component {
+  render() {
+    const tabs =
+      <View style={styles.tabContainer}>
+        <TouchableOpacity
+          style={[styles.tab]}>
+          <StyledText text={ 'QUESTION' } />
+        </TouchableOpacity>
+        <TouchableOpacity
+          style={[styles.tab, styles.deselectedTab]}>
+          <StyledText text={ 'TUTORIAL' } />
```

Comment:
No it won't show up if there isn't one (that part will be included with the upcoming Tutorial PR).